howto/manage/ams-access.md

```diff
@@ -40,8 +40,8 @@ To use a CA certificate, complete the following steps:
 4. For each client, generate a client key and certificate based on the CA certificate. You should use the same method for this as you did in the first step.
 5. Copy the generated credentials to the client machine:
 
-   - Copy the client certificate to `$HOME/snap/ams/current/client/client.crt`.
-   - Copy the client key to `$HOME/snap/ams/current/client/client.key`
+   - Copy the client certificate to `$HOME/snap/amc/current/client/client.crt`.
+   - Copy the client key to `$HOME/snap/amc/current/client/client.key`
```

Comment:
Actually it can be both. If you're using the `ams` snap (which includes amc), then you will have `$HOME/snap/ams`. When you use the appliance, where the `amc` snap is being used, you will have `$HOME/snap/amc`

Reply:
From the context of this page, the `amc` snap rather than the `ams` snap is chosen to be installed. For the appliance, we have amc configured well for end-user during the bootstrap hence they don't need to create keypair by themselves. This page aims to walk through how to access the ams from the `client` machine where the amc snap is going to be installed only.